sql/5_0_0-to-5_0_1_upgrade.sql

```diff
@@ -510,4 +510,17 @@ INSERT INTO `supported_external_dataloads` (`load_type`, `load_source`, `load_re
 
 #IfColumn x12_partners x12_version
 ALTER TABLE `x12_partners` DROP COLUMN `x12_version`;
-#EndIf
\ No newline at end of file
+#EndIf
+
+#IfMissingColumn claims submitted_claim
+ALTER TABLE `claims` ADD COLUMN `submitted_claim` TEXT NULL DEFAULT NULL COMMENT 'This claims form claim data';
+#EndIf
+
+#IfMissingColumn billing revenue_code
+ALTER TABLE `billing` ADD COLUMN `revenue_code` varchar(6) NOT NULL DEFAULT "" COMMENT 'Item revenue code';
+#EndIf
+
+#IfMissingColumn codes revenue_code
+ALTER TABLE `codes` ADD COLUMN `revenue_code` varchar(255) NOT NULL DEFAULT "" COMMENT 'Item revenue code';
```

Comment:
Because in codes I had plans to also write the revenue code description with the code. I'm not sure if that is still not a possibility. May as well keep at 6, Just keeping option open.